core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -18,15 +18,25 @@ import { CONFIGURATIONS } from '../../src/config/configuration';
 import { Timeouts, Browser } from 'selenium-webdriver';
 import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG } from 'constants';
 
+const twok = 2000;
+const fivek = 5000;
+const tenk = 10000;
+const fifteenk = 15000;
+const twentyk = 20000;
+const thirtyk = 30000;
+const sixtyk = 60000;
+
```

Comment:
Good start, at least we have everything in one place. Have logical names and corresponding values. Create different names based on what makes sense in the context of testing the application.

eg. longWait = 60000, mediumWait = 30000 OR 
waitForAPI = 30000, waitForScriptProcess = 50000, etc.

Reply:
i agree. but we will not be able to make these changes in this release. We are still aggregating all sleep times and once we are fairly certain on the different groups we will make the changes as suggested